restapi.go

```diff
@@ -2209,6 +2209,19 @@ func (s *Session) MessageReactionsRemoveAll(channelID, messageID string) error {
 	return err
 }
 
+// MessageReactionsRemove deletes all reactions of a certain emoji from a message
+// channelID : The channel ID
+// messageID : The message ID
+// emojiID	 : The emoji ID
+func (s *Session) MessageReactionsRemove(channelID, messageID, emojiID string) error {
```

Comment:
Since `MessageReactionsRemovaAll` (above) is for

> **Delete All Reactions**
>
>`DELETE/channels/{channel.id}/messages/{message.id}/reactions`
>
>Deletes all reactions on a message. This endpoint requires the 'MANAGE_MESSAGES' permission to be present on the current user.

I think this should be `MessageReactionsRemoveEmoji`:

> **Delete All Reactions for Emoji**
>
> `DELETE/channels/{channel.id}/messages/{message.id}/reactions/{emoji}`
>
> Deletes all the reactions for a given emoji on a message. This endpoint requires the MANAGE_MESSAGES permission to be present on the current user.

```suggestion
func (s *Session) MessageReactionsRemoveEmoji(channelID, messageID, emojiID string) error {
```